nodefs/simple_test.go

```diff
@@ -492,6 +493,39 @@ func TestNotifyEntry(t *testing.T) {
 	}
 }
 
-// Test Notify() , but requires KEEP_CACHE.
+// XXX Test Notify() , but requires KEEP_CACHE ? or could use mmap?
+// XXX Test NotifyDelete?
 
-// Test NotifyDelete?
+func TestReadDir(t *testing.T) {
+	tc := newTestCase(t)
+	defer tc.Clean()
+
+	// XXX what about ".." and "." ?
+	want := map[string]bool{}
+	for i := 0; i < 2; i++ {
```

Comment:
Typo, should be 110 instead of 2?

However, increasing to 110 fails in tests:
```
$ go test
20:12:20.916028 killing subprocess
--- FAIL: TestReadDir (0.02s)
    simple_test.go:517: ReadDir: readdirent: input/output error
FAIL
exit status 1
FAIL	github.com/hanwen/go-fuse/nodefs	0.448s
```

Reply:
there is something wrong with the encoding or size (the failing read is exactly 4096 bytes. if we stop one entry earlier it succeeds.)

Reply:
fixed it.